lib/LANraragi/Utils/Database.pm

```diff
@@ -41,6 +41,7 @@ sub add_archive_to_redis ( $id, $file, $redis ) {
 
     $redis->hset( $id, "name", redis_encode($name) );
     $redis->hset( $id, "tags", "" );
+    set_arcsize( $id, -s $file );
```

Comment:
I'd add a call to `set_arcsize` to `change_archive_id` in this module as well - This should keep size calculations mostly accurate even for the nutjobs who update their archives in-place.

Reply:
(also wow Utils::Database is getting long I should probably split some of it to a Models::Database at some point)

Reply:
Fixed!